config/environments/development.rb

```diff
@@ -34,21 +36,15 @@
   # Print deprecation notices to the Rails logger.
   config.active_support.deprecation = :log
 
-  # Raise an error on page load if there are pending migrations.
-  # config.active_record.migration_error = :page_load
-
-  # Debug mode disables concatenation and preprocessing of assets.
-  # This option may cause significant delays in view rendering with a large
-  # number of complex assets.
-  # config.assets.debug = true
-
-  # Suppress logger output for asset requests.
-  # config.assets.quiet = true
-
-  # Raises error for missing translations
+  # Raises error for missing translations.
   # config.action_view.raise_on_missing_translations = true
 
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.hosts = [
+    "draft-router-api.dev.gov.uk",
+    "router-api.dev.gov.uk",
+  ]
```

Comment:
I think on reflection we should use `<<` so we preserve `127.0.0.1`, `::1` and `localhost` as valid hosts, e.g

``` ruby
config.hosts << "draft-router-api.dev.gov.uk"
config.hosts << "router-api.dev.gov.uk"
```

Reply:
true

Reply:
done